src/Sylius/Bundle/ChannelBundle/Resources/config/services.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services
+                               http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <parameters>
+        <parameter key="sylius.form.type.channel.class">Sylius\Bundle\ChannelBundle\Form\Type\ChannelType</parameter>
+        <parameter key="sylius.context.channel.class">Sylius\Component\Channel\Context\ChannelContext</parameter>
+    </parameters>
+
+    <services>
+        <service id="sylius.form.type.channel" class="%sylius.form.type.channel.class%">
```

Comment:
Both a form types should not be declared as a service but the resource bundle does it.